src/views/settings/filters/parts/create-outgoing-filter-modal.tsx

```diff
@@ -107,7 +108,59 @@ const CreateOutgoingFilterModal: FC<ComponentProps> = ({
 		[activeFilter, filterName, condition, requiredFilterTest, finalActions, dontProcessAddFilters]
 	);
 
-	const createFilterDisabled = useMemo(() => filterName.length === 0, [filterName]);
+	const createFilterDisabled = useMemo(() => {
```

Comment:
this function is exactly the same as used in create-filter-modal.tsx, maybe it can be exported in utils? or maybe in commons in case it is used also in other modules?